janusgraph-server/src/test/java/org/janusgraph/graphdb/grpc/JanusGraphGrpcServerBaseTest.java

```diff
@@ -51,7 +56,7 @@ protected static GraphManager getGraphManager() {
         return new DefaultGraphManager(settings);
     }
 
-    public void createVertexLabel(String graph, VertexLabel vertexLabel) {
+    public void createVertexLabel(String graph, VertexLabelOrBuilder vertexLabel) {
```

Comment:
Just curious: how does this "...OrBuilder" work?

Reply:
vertex can be VertexLabel or VertexLabel.Builder, you don´t need to call build to access read functions.